### DISCUSSION TITLE
without downloading

### DISCUSSION THREAD

**Discussion by zhaibin** [sparkle-project/Sparkle#2565] [Q&A]
<img width="254" alt="image" src="https://github.com/sparkle-project/Sparkle/assets/4013028/f4d9a9db-2a02-47c3-8456-73da8ce2383f">

**When I click the OK button, the dialog disappears without downloading.** 

Here's my appcast.xml. 

`<?xml version="1.0" encoding="utf-8"?>
<rss version="2.0" xmlns:sparkle="http://www.andymatuschak.org/xml-namespaces/sparkle" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>CloudWallpaper Updates</title>
    <link>https://zhaibin.github.io/92CloudWallpaper_MacOS/updates/appcast.xml</link>
    <description>Most recent changes with links to updates.</description>
    <language>zh_CN</language>
    <item>
      <title>Version 0.2.6</title>
      <sparkle:version>0.2.6</sparkle:version>
      <sparkle:releaseNotesLink>https://github.com/zhaibin/92CloudWallpaper_MacOS/releases/download/v0.2.6/CloudWallpaper.zip</sparkle:releaseNotesLink>
      <pubDate>Thu, 23 May 2024 06:52:57 +0000</pubDate>
      <enclosure url="https://github.com/zhaibin/92CloudWallpaper_MacOS/releases/download/v0.2.6/CloudWallpaper.zip"
                 length="1681188"
                 type="application/octet-stream"
                 sparkle:edSignature="/dQJBxZFbBZKrQgmh1NytNTDbUJi1DwznAmmH2Sf/Sh3nMgEZ/OrOjIXC0BB2WUhSli8M/HZIexgDYQkCnUIBw=="/>
    </item>
  </channel>
</rss>`

**Comment by zorgiepoo** [comment]
The CFBundleVersion is probably not set up correctly with the `sparkle:version`. CFBundleVersion corresponds to `sparkle:version`. CFBundleShortVersionString corresponds to `sparkle:shortVersionString`. Sparkle uses `sparkle:version` for comparison, and `sparkle:shortVersionString` for display, similar to how CFBundleVersion and CFBundleShortVersionString are used in an app.

https://sparkle-project.org/documentation/#5-publish-your-appcast
https://sparkle-project.org/documentation/publishing/#internal-build-numbers
